docker/m3coordinator/Dockerfile

```diff
@@ -21,7 +21,7 @@ LABEL maintainer="The M3DB Authors <m3db@googlegroups.com>"
 EXPOSE 7201/tcp 7203/tcp
 
 COPY --from=builder /go/src/github.com/m3db/m3db/bin/m3coordinator /bin/
-COPY --from=builder /go/src/github.com/m3db/m3db/src/coordinator/config/m3coordinator-local-etcd.yml /etc/m3coordinator/m3coordinator.yml
+COPY --from=builder /go/src/github.com/m3db/m3db/src/query/config/m3coordinator-local-etcd.yml /etc/m3coordinator/m3coordinator.yml
```

Comment:
hm do you want to chance the directory name here too

Reply:
Yeah maybe we should move these config files outside of `src` completely?  It's weird we have some YAML files in these dirs floating around.

Reply:
i guess we should figure out if we want to have an m3coordinator and an m3query dockerfile? we can discuss today once everyone is in